packages/core/src/routes/admin-user.ts

```diff
@@ -264,4 +266,25 @@ export default function adminUserRoutes<T extends AuthedRouter>(router: T) {
       return next();
     }
   );
+
+  router.delete(
+    '/users/:userId/identities/:connectorId',
+    koaGuard({ params: object({ userId: string(), connectorId: string() }) }),
+    async (ctx, next) => {
+      const {
+        params: { userId, connectorId },
+      } = ctx.guard;
+
+      const { identities } = await findUserById(userId);
+
+      if (!has(identities, connectorId)) {
+        throw new RequestError({ code: 'user.identity_not_exists', status: 404 });
```

Comment:
nit: I believe default error status is 404. Could use `assertThat(has(identities, connectorId), 'user.identity_not_exists')`

Reply:
RequestError default status is 400, need to specify.